### ISSUE TITLE
Vet Center incorrect heading structure in accordions

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#18242]
DECISION 3-26-25: Alert to be removed. This work will get its own ticket.

## Description
At the bottom of all [Vet Center pages](https://www.va.gov/boston-vet-center/), there are 2 heading structure issues that should be resolved:

1. The first issue is that all accordion titles should be h3s but the "How we are different" section has accordions that are h2s:
![Screenshot 2024-05-29 at 8 44 00 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/039087a8-03fb-4c58-a64c-1f985420d158)
2. Second issue is, within this particular accordion, there is an alert that has an h2 heading. This should actually be an h4:
![Screenshot 2024-05-29 at 8 45 21 AM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/7c9c4a43-554b-4519-9532-7f4dd9bddb17)

## Acceptance Criteria
- [ ] Update the heading of the "Am I eligible for Vet Center services as a Veteran or service member?" accordion to be an H3
- [ ] Reformat the content so that it is normal text instead of an alert nor has a heading level.
- [ ] Requires accessibility review

**Comment by Agile6MSkinner** [comment]
I'm moving this back to the backlog because it appears that there is design work that still needs to be done before we can point it.

**Comment by laflannery** [comment]
@mmiddaugh and possibly @aklausmeier In the ticket description I have a question that I think you might be able to provide insight on, I have provided the info/question here as well:

On each Vet center page, in the "How are we different" section, there is an accordion for "Am I eligible for Vet Center services as a Veteran or service member?". This accordion content has an alert with a header level defect currently. However I am curious if we even need to have this content in an alert. It's a simple bullet list and by adding this content in an additional info component within an alert just adds complexity that I don't think we need. 

Do you know if we are able to remove this format and simply add the bullets within the content instead?

**Comment by aklausmeier** [comment]
I am in favor of pulling content out of hidden additional info components and don't see a need for the alert. Unless @mmiddaugh has historical context/prior decisioning for keeping this.

**Comment by jilladams** [comment]
The current body of this ticket is isolated to 2 things: 
1. Fixing accordion headers to become h3
2. Changing the alert header within the eligibility accordion from h2 to h4

Comments from Laura / Amanda in ticket say: 
* Remove the alert within that accordion, and convert that content to become bullets, unless @mmiddaugh objects. 

I don't see any design calls that need to be made. @mmiddaugh if you can advise about the Alert, we can update ticket and refine this. Moving to Backlog until we get that clarified. 

Also noting: this ticket isn't currently in an epic. I don't think it makes sense to tack it onto Vet Center 2.0, so didn't do that.

**Comment by laflannery** [comment]
@mmiddaugh I noticed a couple things which make me continue to lean towards removing this current display and just add bullets:
- In the original content, there were 9 bullets in the proposed content ut we now only have 5. That's definitely a large difference in the amount of content and the amount of space that they take up
- It's very clear that an "Additional info" component was proposed by CAIA, not an "Alert WITH an additional info" component. The reason it was done the way it was is because of limitations within Drupal, they aren't able to only add the additional info component here. (Having just an additional info component here wouldn't pose the same heading issues as we are seeing now)

My recommendation is that we remove this display, replace with simple text and a bullet list and if in future we keep seeing a need for the additional info component then we can approach it from the Drupal side. Let me know if that makes sense!

**Comment by randimays** [comment]
@Agile6MSkinner This would be a fairly simple FE ticket if we need something to refine/work. To clarify:

- FE code (content-build) changes: Changing the accordion header level
- Drupal content changes: Removing the alert and making it plain bulleted text (anybody can do this at https://prod.cms.va.gov/centralized-content/national-vet-center-content)

Added points.

**Comment by davidmpickett** [comment]
@Agile6MSkinner I drafted a change to the Vet Center centralized content to reformat the content inside the accordion to not use an alert. I'll leave it to you to confirm and publish

https://prod.cms.va.gov/node/16142/latest

**Comment by laflannery** [comment]
Draft content published, this is solved. The accordion level issue was caused because of the alert as well
